try {
    functionExample()
} catch(error) {
    //console.log(error)
    console.log("this is a custom message")
}